CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-08-06
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* For AKS clusters built at version v1.27+ and enable KMS, KMS v2 is configured by default. However, for clusters with KMS enabled at versions below v1.27, upgrading to v1.27 will be blocked. To upgrade, follow the steps outlined in this [documentation](https://learn.microsoft.com/azure/aks/use-kms-etcd-encryption#migration-to-kms-v2) for migrating from KMS v1 to v2, and then proceed with upgrading the cluster to version v1.27.
+* The pod security policy feature was deprecated on 1st August 2023 and removed since AKS version 1.25. We recommend you migrate to [pod security admission controller](https://learn.microsoft.com/azure/aks/use-psa) or [Azure Policy](https://learn.microsoft.com/azure/aks/policy-reference) to stay within Azure support.
+
+### Release notes
+
+* Behavioral changes
+  * API structure for AutoscalerProfile has been updated.
+  * Customers will now be able to use node public IP with authorized IP ranges and APIservervnet integration. Previously this functionality was blocked.
+  * Customers can now install Azure Service Mesh with on AKS clusters with Cilium.
+  * Configure exponential backoff in calls from the Cilium daemonset to the Kubernetes apiserver in Azure CNI Powered by Cilium to improve recovery from OOM kills. 
+
+ * Bug Fixes
```

Comment:
Same comment as above - Lets put GH links or public docs link on the features being spoken about , the changes may not be there in the public doc but reference is key.

Reply:
Updated - please feel free to provide feedback on the additional context added